test/utils.ts

```diff
@@ -0,0 +1,133 @@
+/* env jest */
+import path from 'path'
+import webpack from 'webpack'
+import merge from 'webpack-merge'
+// import MiniCssExtractPlugin from "mini-css-extract-plugin";
+import { fs as mfs } from 'memfs'
+
+import { VueLoaderPlugin, VueLoaderOptions } from '../src/index'
+
+const baseConfig: webpack.Configuration = {
+  mode: 'development',
+  devtool: false,
+  output: {
+    path: '/',
+    filename: 'test.build.js'
+  },
+  resolve: {
+    alias: {
+      // this isn't technically needed, since the default `vue` entry for bundlers
+      // is a simple `export * from '@vue/runtime-dom`. However having this
+      // extra re-export somehow causes webpack to always invalidate the module
+      // on the first HMR update and causes the page to reload.
+      vue: '@vue/runtime-dom'
+    }
+  },
+  resolveLoader: {
+    alias: {
+      'vue-loader': require.resolve('../src')
```

Comment:
Does this work because jest registers require hooks and compiles the ts src for all require calls? I think we should probably run build before test and run the test on the build dist files since that is closer to production.

Reply:
Done